vertical-pod-autoscaler/pkg/recommender/model/cluster_test.go

```diff
@@ -166,7 +208,7 @@ func TestAddVpaThenAddPod(t *testing.T) {
 	addTestPod(cluster)
 	container := addTestContainer(cluster)
 	aggregateStateKey := cluster.aggregateStateKeyForContainerID(testContainerID)
-	assert.True(t, container.aggregator == vpa.aggregateContainerStates[aggregateStateKey])
+	assert.True(t, container.aggregator.Get() == vpa.aggregateContainerStates[aggregateStateKey])
```

Comment:
IMO this Get metod should be deleted:
- it is not obvious what it returns and why it should be part of the interface
- it is used only in tests
I recommend testing with Contains/NotContains(t, vpa.aggregateContainerStates, aggregateStateKey)

Reply:
You're right, I was just being lazy. Removed.